tests/test_csc.py

```diff
@@ -157,6 +151,10 @@ async def doit():
                 self.assertEqual(id_ack.error, 0)
                 self.assertEqual(harness.csc.summary_state, salobj.State.DISABLED)
                 self.assertEqual(state.summaryState, salobj.State.DISABLED)
+                # Verify mirror is NOT lowered when transitioning from STANDBY
```

Comment:
Not sure I understand this test here. Why asserting that `harness.pnematics.cmd_m1SetPressure.callback` was not called verify the mirror is not lowered when transitioning from standby to disabled?

Also, in unit tests it is better to use `self.assertTrue` instead of the python builtin `assert` method. The first is a unit test method that makes sure to register the unit test failure, the later will raise an `AssertionError` exception.

Reply:
I'll use `self.assertTrue` - no problem.

Addressing the comments below here as well since it's all related... but I feel like I'm just not understanding your question. The tests here are essentially verifying that the mirror is only being lowered for the appropriate state transitions. Essentially if the method is called then the mirror gets lowered, if it doesn't then it is left in it's current position.... but I feel like you saw that so I'm pretty sure I'm missing your concern...